scans/auth/plans_and_scripts/dev_bba.yaml

```diff
@@ -0,0 +1,48 @@
+env:
+  contexts:
+  - name: BBA_Test
+    urls:
+    - http://localhost:9091/auth/simple-json-cookie/
```

Comment:
The dev auth pages we test elsewhere - these tests should just be for online sites

Reply:
I know, needed something to use to get it setup.